samples/modules/thrift/hello_common/native-cert.pem

```diff
@@ -0,0 +1,22 @@
+"-----BEGIN CERTIFICATE-----\n"
+"MIIDkzCCAnugAwIBAgIUaGlOEAH7p7FyprSNEHTH3Yy3+8owDQYJKoZIhvcNAQEL\n"
```

Comment:
Ah, now I see... technically, this is not a PEM file, I don't think I like this

Reply:
I would guess I missed this somehow as well from our GSoC contributor.